projects/sm-pwr/bop/steamer.py

```diff
@@ -476,7 +508,8 @@ def __heat_sink_rate(self, water_p, water_s):
         #############################################
         radius_inner = self.helicoil_inner_radius
 
-        temp_s_w = temp_p_w - self.wall_temp_delta_secondary
+        #temp_s_w = temp_p_w - self.wall_temp_delta_secondary
+        temp_s_w = temp_p_w
```

Comment:
There should be a small delta T here to account for the thickness of the tube wall.